### PR TITLE
Added Code Monk Series and competitive contest code

### PR DESCRIPTION
**Pull request by RahulSurana123**
# Have you read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/CONTRIBUTING.md)?
- [x] YES
- [ ] NO

## Project URL
- 

## What is the current behavior?
- Issue Number: #...

## Category
- [ ] 𝑨𝑺𝑲 𝒀𝒐𝒖𝒓 𝑫𝒐𝒖𝒃𝒕𝒔
- [ ] 𝑴𝒊𝒔𝒄𝒆𝒍𝒍𝒂𝒏𝒆𝒐𝒖𝒔 𝑺𝒕𝒖𝒇𝒇.
- [ ] 𝑫𝑺𝑨
- [x] 𝑪𝒐𝒎𝒑𝒆𝒕𝒊𝒕𝒊𝒗𝒆 𝑷𝒓𝒐𝒈𝒓𝒂𝒎𝒎𝒊𝒏𝒈
- [ ] 𝑻𝒆𝒄𝒉𝒏𝒊𝒄𝒂𝒍 𝑺𝒖𝒃𝒋𝒆𝒄𝒕
- [ ] 𝑳𝒐𝒘 𝑳𝒆𝒗𝒆𝒍 𝑫𝒆𝒔𝒊𝒈𝒏
- [ ] 𝑷𝒓𝒐𝒋𝒆𝒄𝒕𝒔
- [ ] 𝑰𝒎𝒑𝒐𝒓𝒕𝒂𝒏𝒕 𝑩𝒐𝒐𝒌𝒔 𝒂𝒏𝒅 𝑹𝒆𝒔𝒐𝒖𝒓𝒄𝒆𝒔
- [ ] 𝑩𝒆𝒉𝒂𝒗𝒊𝒐𝒖𝒓𝒂𝒍 𝑰𝒏𝒕𝒆𝒓𝒗𝒊𝒆𝒘 𝑸𝒖𝒆𝒔𝒕𝒊𝒐𝒏𝒔

## Changes
- Description of changes

## **Screenshots** 
- (Add Runnig Work Demo Image)

## Checklist
- [x] I've read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/CONTRIBUTING.md).
- [x] I've checked the issue list before deciding what to submit.
- [x] I've follow [Tree of Index](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation#tree-of-index).
- [x] I've add work in proper folder
- [x] I've not delete/modify previous codes.

### CODE REVIEW

**Comment by RahulSurana123** [comment]
Added Code Monk Series and Competitve Contest Codes

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
check code again
attach accepted code screenshot in PR Template

Also, do proper numbering and make folder 

image

see the image and make folders like `.../ 1]. Practice/5]. Code Monk/01]. Arrays & Strings/03). Cyclic shift.cpp `

File: 3]. Competitive Programming/06]. CodeChef/2]. Compete/2]. Cook-off/Div 3/The Wave.cpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/06]. CodeChef/2]. Compete/2]. Cook-off/Div 3/The Wave.cpp:1]: Accepted
But Add month
like `.../Div 3/June/3). The_Wave.cpp`

Comment (RahulSurana123) [reply 3]. Competitive Programming/06]. CodeChef/2]. Compete/2]. Cook-off/Div 3/The Wave.cpp:1]: Done

File: 3]. Competitive Programming/05]. HackerEarth/1]. Practice/5]. Code Monk Series/1] Arrays & Strings/Cyclic shift.cpp [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+First line: A single integer T denoting the number of test cases
+For each test case:
+First line: Two space-separated integers N and K 
+Second line: A denoting the string
+
+    ***********************************************************
+
+*/
+
+#include <bits/stdc++.h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/05]. HackerEarth/1]. Practice/5]. Code Monk Series/1] Arrays & Strings/Cyclic shift.cpp:30]: check this code again
image

File: 3]. Competitive Programming/05]. HackerEarth/1]. Practice/5]. Code Monk Series/5] Number Theory Part 1/Monk and Divisor Conundrum.cpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+The next line contains a single integer T denoting the number of queries Monk poses to you. Each of the next T lines contains 2 space separated integers P and Q.
+
+Output Format :
+
+For each query, print the answer on a new line.
+
+
+    ***************************************************************************
+
+*/
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/05]. HackerEarth/1]. Practice/5]. Code Monk Series/5] Number Theory Part 1/Monk and Divisor Conundrum.cpp:27]: check this 

no code

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Pull request merged**